### DISCUSSION TITLE
Vertical MultiStepIndicator

### DISCUSSION THREAD

**Discussion by ramarm** [mondaycom/vibe#1987] [Ideas]
Hey vibe team,
I would be happy if the multi step indicator would be supported verticaly.
We have a use case that we need the steps intecators to be on the left side of the screen when there is the step content on the left.
Here is for example what I mean that supported in the Ant Design - https://ant.design/components/steps#steps-demo-vertical

Thank you very much!

**Comment by talkor** [comment]
Hey @ramarm! Thank you for this suggestion! We won't be able to introduce this specific feature in the near future, but in the meantime you might be able to create something similar by composing other Vibe components.
Thanks!
